README.md

```diff
@@ -0,0 +1,14 @@
+```
```

Comment:
How about some installation notes:

````suggestion
## Installation

The script is available  on `PyPI`:

```
pip install markdown-command
```
````

?. I think I would add that between `Process a Markdown` and `Usage` part. Does it make sense?